.tekton/creatctconfig-0-6-pull-request.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "redhat-v0.6"
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "push" && target_branch == "redhat-v0.6" && ( "cmd/ctlog/createctconfig/*".pathChanged() || ".tekton/createctconfig-*-pull-request.yaml".pathChanged() || "/Dockerfile.createctconfig".pathChanged() || "Makefile".pathChanged() || "go.mod".pathChanged() || "config/ctlog/certs/*".pathChanged())
```

Comment:
Not sure if we need to check the `config/ctlog/certs/*` path was hoping someone could weigh in on this.